components/product/SpecialNotice.js

```diff
@@ -1,12 +1,12 @@
 import React from 'react';
-import { Notice } from '../../styled/product';
+import { Body } from '../BaseComponents';
 
 function SpecialNotice() {
   return (
-    <Notice>
+    <Body>
```

Comment:
This is a Caption, not Body

Reply:
Color is secondaryText

Reply:
SpecialNotice deleted and moved to ProductInfo.js in #34. Will implement changes as part of that PR!